udf-compiler/src/test/scala/com/nvidia/spark/OpcodeSuite.scala

```diff
@@ -2086,4 +2086,202 @@ class OpcodeSuite extends FunSuite {
     val ref = dataset.select(lit(Array.empty[String]).as("emptyArrOfStr"))
     checkEquiv(result, ref)
   }
+
+  test("final static method call inside UDF - ") {
```

Comment:
minor nit, there's a ` - ` at the end of some of these tests. Is that on purpose?

Reply:
Removed the trailing `-`